tests/test_doctests.py

```diff
@@ -57,39 +59,46 @@
 suite = unittest.TestSuite()
 for mod in mods:
     try:
-        suite.addTest(doctest.DocTestSuite(mod))
+        pass
+        #suite.addTest(doctest.DocTestSuite(mod))
```

Comment:
Yes, they are currently excluded as they will fail when not run from a local context